File: testgo/Basic/Str/Substr_19.go
package Str

import (
     "libs/helper/go/Basic"
    "log"
    )

func TestSubstr_19(){
    news :=Basic.NewStr("abc").Substr(1,1)
    if news!="b"{
        log.Fatal(news)
    }

    news =Basic.NewStr("abcd").Substr(1,10)
    if news!="bcd"{
        log.Fatal(news)
    }
    log.Print("OK")
}

File: test/Basic/Str/Substr_19.php
<?php

namespace xltxlm\helper\test\Basic\Str;

use xltxlm\helper\Basic\Str;

/**
 *
 */
class Substr_19
{

    public function __invoke()
    {
        $news = (new Str("abc"))
            ->Substr(1, 1);
        if ($news != 'b') {
            throw new \Exception($news);
        }

        $news = (new Str("abcd"))
            ->Substr(1, 10);
        if ($news != 'bcd') {
            throw new \Exception($news);
        }


    }

}

